Feature #33 - show archived bundles

ameer-clara opened the pull request:
- feed upload payload schema matches interface in browse-page
- bundle payload schema updated to include archiveItems and mediaMap
- convert profile image to ascii art and add to feed
- update avatar-card to support archived bundles and pending backup
- add new actions (get, load, fail) for handling feed downloads
- show last 5 archived bundles

kirlen reviewed (changes requested):
freakin' awesome, ameer!

ameer-clara commented on client/src/utils/store.js:
```diff
@@ -203,14 +236,47 @@ const StoreProvider = ({ children }) => {
+  const downloadFeedsFromSwarm = async () => {
+    let b = new Beejs();
+    let feedIndex = await b.getFeedIndex();
+    let feeds = await b.getFeeds(feedIndex, 5);
```
Good catch! Should come from front end. Will update GET_FEEDS_FROM_SWARM dispatch and browse page to support it. After this merge, we should support pagination and some sort of cache as @codiak mentioned in our call.

ameer-clara commented on client/src/process/Beejs.tsx:
```diff
@@ -48,26 +61,142 @@ export class Beejs {
+
+  trackRequest(progressCb: any, upload: boolean) {
+    const axiosInstance = upload ? axios.create({ onUploadProgress: progressCb }) : axios.create({ onDownloadProgress: progressCb });
+    return buildAxiosFetch(axiosInstance);
```
Thanks, much cleaner, noted.

kirlen replied:
👍

kirlen commented:
@ameer-clara, since you keep tweaking (improving!) this branch, had to join in the fun by displaying the archive date in there. hope you don't mind!

kirlen reviewed (approved):
When loading feeds from cache, UI still says "Downloading", and takes longer than I'd expect -- but approving PR because clearly you will keep tweaking until it's merged ;-) !!!

ameer-clara commented:
> When loading feeds from cache, UI still says "Downloading", and takes longer than I'd expect -- but approving PR because clearly you will keep tweaking until it's merged ;-) !!!

Thanks!!!! Updated #62 with sequence diagrams to show caching and implemented an improvement.
